content/blog/2020/12/2020-12-05-gsod-project-report.adoc

```diff
@@ -0,0 +1,117 @@
+I also made sure to use updated terms and terminologies where necessary like Controller instead of Master and Agents instead of Slave. 
+
+During the documentation phase, I was able to work on documenting link:https://www.jenkins.io/doc/book/installing/kubernetes/[Installing Jenkins on Kubernetes] with three sections link:https://www.jenkins.io/doc/book/installing/kubernetes/#install-jenkins-with-helm-v3[Helm], link:https://www.jenkins.io/doc/book/installing/kubernetes/#install-jenkins-with-yaml-files[Set of Yaml files] and link:https://www.jenkins.io/doc/book/installing/kubernetes/#install-jenkins-with-jenkins-operator[Jenkins Operator]. 
+I also worked on creating a directory for Jenkins on kubernetes sample files in Jenkins.io repository, documenting link:https://github.com/jenkins-infra/jenkins.io/pull/3979[Scaling Jenkins on Kubernetes] and Jenkins on AWS which is still in progress.
+
+
+## Work Done
+
+**Pull Requests**: All the pull requests I submitted to Jenkins.io documentation can be found link:https://github.com/jenkins-infra/jenkins.io/pulls?q=is%3Apr+author%3Azaycodes[here]. 
+This link:https://docs.google.com/spreadsheets/d/1Jvu9HkWmNycjMkGxUkgCQXhkgX4gzvTQsFn7i7c9NUA/edit?usp=sharing[spreadsheet] contains links to the published documentation on link:https://www.jenkins.io/[Jenkins.io]. 
+The link:https://docs.google.com/spreadsheets/d/1Jvu9HkWmNycjMkGxUkgCQXhkgX4gzvTQsFn7i7c9NUA/edit?usp=sharing[spreadsheet] also highlights the initial proposed tasks and the status of each of them.
+
+If you would like to contribute to the Jenkins on Kubernetes documentation, you can check out pending tasks link:https://docs.google.com/spreadsheets/d/1Jvu9HkWmNycjMkGxUkgCQXhkgX4gzvTQsFn7i7c9NUA/edit?usp=sharing[here] and reach out in the Jenkins documentation link:https://gitter.im/jenkinsci/docs[gitter] channel.
+
+
+## Challenges
+
+Using a Windows pc was a bit of a challenge for me. 
+To run Jenkins.io locally, the project uses GNU/Make and Docker in order to generate the fully statically generated link:https://www.jenkins.io/[jenkins.io] web site. 
+The key tool for converting source code into the site is the link:https://github.com/awestruct/awestruct[Awestruct] static site generator, which is downloaded automatically as part of the build process. 
+To achieve this, I needed to have GNU/Make and Docker available on my machine. 
+Docker was not a problem, but to achieve the latter, I needed to use link:https://docs.microsoft.com/en-us/windows/wsl/install-win10[Windows Subsystem for Linux] (WSL). 
+WSL had two versions WSL1 and WSL2. Using WSL2 would have been much more convenient, but my version of windows wasn’t compatible with WSL2 as it required Version 1903 or higher, with Build 18362 or higher for x64 systems. 
+With this obstacle, I had to stick to making WSL1 work but I still couldn’t get this to work, a series of issues came up which I was able to pass through with the help of my mentors until I got stuck at permission issues. 
```

Comment:
Ugh. This is something I could have helped with :( Sorry for not noticing it in the chat

Reply:
Thank you. I'm glad I was able to get past it at the end of the day. Although I would still love to find a way to make WSL 1 work